ha/virsh/tests/fence_ipmilan_test.sh

```diff
@@ -52,8 +52,11 @@ get_tester_ip() {
 }
 
 setup_tester() {
-  run_command_in_node "${IP_TESTER}" "sudo apt-get update && \
-    sudo apt-get install -y ipmitool fence-agents-base"
+  APT_OUTPUT=$(run_command_in_node "${IP_TESTER}" "DEBIAN_FRONTEND=noninteractive sudo apt-get update && sudo apt-get install -y ipmitool fence-agents-base")
+  if [ $? -ne 0 ]; then
```

Comment:
Is this `if` needed because we want to explicitly return 1 when `$?` is nonzero for some reason?

Reply:
I see, but I the function should return the RC of the latest command:
```
$ foo() { (exit 42) }
$ foo
$ echo $?
42
```
so I'm still not sure of why that `if` is needed...

Reply:
I was changing the error status code to 1.

Let's preserve it then. Fixed